.github/workflows/temporary-parallel-review.yml

```
@@ -0,0 +1,90 @@
+name: Temporary Parallel PR Review
+
+run-name: ${{ format('Parallel review of PR #{0}', inputs.pr_number || vars.REVIEW_CAMPAIGN_PR_NUMBER || '252') }}
```

actionlint (1.7.12)

[error] got unexpected EOF while lexing end of string literal, expecting '''

(expression)